src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/data/ActivityMappingDto.kt

```diff
@@ -28,15 +28,11 @@ data class ActivityMappingDto(
   @Schema(description = "Date-time the mapping was created")
   val whenCreated: LocalDateTime? = null,
 
-  @Schema(description = "Label (a unique reference for migrations)")
-  @field:Size(max = 20)
-  val label: String? = null,
```

Comment:
Removed label (used store the migration id) from the existing activity mapping table as we won't use that table for the migration.